Explorer/Assets/DCL/Chat/Commands/SpamChat/ChatMessageSpammer.cs

```diff
@@ -0,0 +1,89 @@
+using System;
+using Cysharp.Threading.Tasks;
+using DCL.Chat.History;
+using DCL.Chat.MessageBus;
+using DCL.Diagnostics;
+
+namespace DCL.Chat.Commands.SpamChat
+{
+    public class ChatMessageSpammer
+    {
+        private readonly IChatMessagesBus chatMessagesBus;
+        
+        public ChatMessageSpammer(IChatMessagesBus chatMessagesBus)
+        {
+            this.chatMessagesBus = chatMessagesBus;
+        }
+
+        public async UniTaskVoid SpamMessagesAsync(SpamExecutionOptions options)
+        {
+            ReportHub.Log(ReportCategory.CHAT_MESSAGES,
+                $"[Spammer-{options.SpamSessionId}] Starting Spam on Channel '{options.TargetChannel.Id.Id}' (Type: {options.TargetChannel.ChannelType}): " +
+                $"Total={options.TotalMessages}, BatchSize={options.BatchSize}, " +
+                $"MsgDelay={options.DelayBetweenMessagesMs}, BatchDelay={options.DelayBetweenBatchesMs}");
+
+            var messagesSent = 0;
+
+            // NOTE: delay to allow the command to be processed before starting the spam
+            await UniTask.Delay(200, ignoreTimeScale: true, cancellationToken: options.CancellationToken);
```

Comment:
Since this is just for testing, it's fine, but generally, a computer could have a second long hiccup (graphics driver crashed, for example) such that 200 ms pass, but nothing got processed yet. It is rare for a hard coded delay to not be a code smell.